Contest_Archive_Format.md

```diff
@@ -138,17 +140,19 @@ data.
 
 ### Design principles
 
-Endpoints are stored in a single <endpoint>.json file containing the
+Endpoints are stored in a single \<endpoint>.json file containing the
```

Comment:
```suggestion
Endpoints are stored in a single `<endpoint>.json` file containing the
```

Reply:
If we are doing this, we need a lot of place to do this. So should we? Then I will update them all.